app/src/main/java/com/github/polybooks/SaleInformationActivity.kt

```diff
@@ -0,0 +1,45 @@
+package com.github.polybooks
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import com.github.polybooks.core.Sale
+import com.github.polybooks.utils.StringsManip
+
+class SaleInformationActivity: AppCompatActivity() {
```

Comment:
I forgot to do it on my own activities as well (I will modify my own PR), but you need to add some doc in front to explain in two-three words what the activity does.

Reply:
Done :)